pallets/automation-time/src/tests.rs

```diff
@@ -2462,30 +2565,39 @@ fn trigger_tasks_completes_some_xcmp_tasks() {
 					asset_location: MultiLocation::new(0, Here).into(),
 					amount: 10,
 				},
-				encoded_call: vec![3, 4, 5],
+				encoded_call: encoded_call.clone(),
 				encoded_call_weight: Weight::from_ref_time(100_000),
 				overall_weight: Weight::from_ref_time(200_000),
 				schedule_as: None,
 				instruction_sequence: InstructionSequence::PayThroughSovereignAccount,
 			},
+			vec![],
 		);
 
 		LastTimeSlot::<Test>::put((LAST_BLOCK_TIME, LAST_BLOCK_TIME));
 		System::reset_events();
 
 		AutomationTime::trigger_tasks(Weight::from_ref_time(120_000));
 
+		let mut condition: BTreeMap<String, String> = BTreeMap::new();
+		condition.insert(String::from("type"), String::from("time"));
+		condition.insert(String::from("timestamp"), format!("{}", LAST_BLOCK_TIME));
+
 		assert_eq!(
 			events(),
 			[
-				RuntimeEvent::AutomationTime(crate::Event::XcmpTaskSucceeded {
-					destination,
+				RuntimeEvent::AutomationTime(crate::Event::TaskTriggered {
+					who: owner.clone(),
 					task_id: task_id.clone(),
+					condition,
+					encoded_call,
+					abort_errors: vec![],
 				}),
```

Comment:
Is it possible to verify the success event of the encoded_call here?

Reply:
No, this is an XCMP task, and its encoded call will not be executed on turing.

Reply:
Low priority - this doesn’t need to block this PR. I created a https://github.com/OAK-Foundation/OAK-blockchain/issues/395 to address it later.